boilerplate/tag/status-NOTE-WD.include

```diff
@@ -5,7 +5,7 @@
     latest revision of this technical report can be found in the <a
     href="https://www.w3.org/TR/"><abbr title="World Wide Web
     Consortium">W3C</abbr> standards and drafts index</a>
-    at https://www.w3.org/TR/.</em>
+   .</em>
```

Comment:
This results in an extraneous space that cause verification to fail

Reply:
#161 should address this